src/app/comic/create/publish/BranchPlacer.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import {useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import PublishButton from "./PublishButton";
+import {PanelSet, Branch} from "./PanelSet";
+
+
+const BranchPlacer = () => {
+    const [branCount, setBranch] = useState(0);
+    const [adding, setAdding] = useState(false);
+    const [parId, setparId] = useState(" ");
+    const [curId, setCurId] = useState(parId ? parId : uuidv4());
+
+    const baseBranch = {
+        panel: -1,
+        x: 0,
+        y: 0,
+        child_branch_uuid: "string"
+    }
+
+    const startPs = {
+        current_panel_set_uuid: curId,
+        parent_branch_uuid: parId,
+        image_paths: ["","",""],
+        branches:[baseBranch,baseBranch,baseBranch]
+    }
+
+    const [ps, setPs] = useState<PanelSet>(startPs);
+    
+    const startAdd = () => {
+        console.log("You have reached maximum amount of branches");
+        if(branCount < 3 ){
+            console.log("Start adding");
+            setAdding(true);
+        }
+    }
+
+    const addHook = (panel:number) => {
```

Comment:
Rename to addBranchHook, as Hooks are a react thing too.

Reply:
ah i didnt know that but i fixed it so addHook -> addBranchHook & removeHook -> removeBranchHook